common/views/components/DefaultPageLayout/DefaultPageLayout.js

```diff
@@ -62,7 +62,7 @@ export const TwitterCard = ({
   type,
   url,
   title,
-  description,
+  description = '',
```

Comment:
Could we get this?

Reply:
We get it when it exists, but when it doesn't we're still passing it to `striptags()` (which tries to do a string replace). I suppose the better solution is to only conditionally pass it to striptags.